diff --git a/Personal-Dashboard--main/app.js b/Personal-Dashboard--main/app.js
index 1ece058..23031a9 100644
--- a/Personal-Dashboard--main/app.js
+++ b/Personal-Dashboard--main/app.js
@@ -2390,9 +2390,8 @@ window.calcAction = function (type, val) {
     if (calcCurrent === "") {
       if (calcPrevious !== "") {
         calcOperation = val;
-      } else {
-        return;
       }
+      return;
     } else {
       if (calcPrevious !== "") {
         calcCompute(false);
